spacetemplate/assets/legacy.yaml

```diff
@@ -15,14 +15,17 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Task
   can_construct: yes
-  description: TBD
+  description: Work assigned to various team members to implement a feature.
+  They are generally measured in units of 0.5 days, for example, four hours,
+  eight hours, sixteen hours.
   icon: fa fa-tasks
 
 - id: &bugID "26787039-b68f-4e28-8814-c2f93be1ef4e"
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Bug
   can_construct: yes
-  description: TBD
+  description: A defect that causes unexpected behavior in the software and that
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed